src/main/java/frc/robot/subsystems/ElevatorArmSubsystem.java

```diff
@@ -44,16 +45,19 @@ public enum Setpoint {
     kL4;
   }
 
+  // Elevator Motor
   private SparkFlex elevatorMotor =
       new SparkFlex(ElevatorConstants.ELEVATOR_MOTOR, MotorType.kBrushless);
   private SparkClosedLoopController elevatorClosedLoopController =
       elevatorMotor.getClosedLoopController();
+  private RelativeEncoder elevatorEncoder = elevatorMotor.getExternalEncoder();
```

Comment:
```suggestion
  private SparkAbsoluteEncoder elevatorEncoder = elevatorMotor.getAbsoluteEncoder();
```

Maybe doing something like this would grab the absolute encoder?